go/obscuronode/host/host.go

```diff
@@ -291,7 +291,18 @@ func (a *Node) startProcessing() {
 	a.EnclaveClient.Start(lastBlock)
 
 	if a.config.IsGenesis {
-		a.initialiseProtocol(&lastBlock)
+		// if the ContractMgmtBlkHash is not specified (like in the ethereum_mock)
+		// the genesis rollup will use the latest block
+		if a.config.ContractMgmtBlkHash.Hex() == (common.Hash{}).Hex() {
+			a.initialiseProtocol(lastBlock.Hash())
```

Comment:
@otherview Isn't it better to make the contractmgmtblockhash mandatory, and handle this on the ethereum mock side? The Ethereum mock could check if the hash is some specific value, and return the latest block if so.

In general, IMO it's better for the "real" code to not know that mocks could be used, and take a single approach.

Reply:
Ya sure that makes sense 👍